assets/js/app.ts

```diff
@@ -147,41 +147,58 @@ const Hooks = {
     },
   },
   VideoPlayer: {
-    mounted() {
+    async mounted() {
       const backdrop = document.querySelector("#video-backdrop");
-
       this.playerId = this.el.id;
 
       // TODO: remove this once we have a better way to handle autoplay
       const autoplay = this.el.id.startsWith("analytics-") ? false : "any";
 
-      this.player = videojs(this.el, {
-        autoplay: autoplay,
-        liveui: true,
-        html5: {
-          vhs: {
-            llhls: true,
-          },
-        },
+      this.player = await window.VidstackPlayer.create({
+        target: this.el,
+        autoplay,
```

Comment:
Autoplay unfortunately doesn't work if the user did not interact with the page yet. Let's handle that scenario with the `onAutoPlayFail` callback, such that we request autoplay again but this time after muting the video (which should always work)

Reply:
Okay. Coool. I'll add this too.